calendar-bundle/contao/modules/ModuleEventReader.php

```diff
@@ -179,7 +179,7 @@ protected function compile()
 
 				$htmlHeadBag->setCanonicalUri($url);
 			}
-			elseif ($objCalendar->enableCanonical)
+			elseif (!$objCalendar->cal_keepCanonical)
```

Comment:
As the setting moved to the reader module, it is not accessiable via `$objCalendar`, right?
```suggestion
			elseif (!$this->cal_keepCanonical)
```